core/build.rs

```diff
@@ -19,13 +19,19 @@
 // DEALINGS IN THE SOFTWARE.
 
 fn main() {
-    prost_build::compile_protos(
-        &[
+    let cust: protobuf_codegen::Customize = Default::default();
+    protobuf_codegen::Codegen::new()
+        .pure()
+        .includes(&["src"])
+        .inputs(
+            &[
             "src/keys.proto",
             "src/envelope.proto",
             "src/peer_record.proto",
-        ],
-        &["src"],
-    )
-    .unwrap();
+            ]
+        )
+        .customize(cust.lite_runtime(true))
```

Comment:
All the builder fns take `self` so I think we can just do this:

```suggestion
        .customize(protobuf_codegen::Customize::default().lite_runtime(true))
```